.github/workflows/ccip-integration-test.yml

```diff
@@ -32,10 +32,20 @@ jobs:
           cd ccip
           git fetch
           git checkout ccip-develop
```

Comment:
Why not use the `actions/checkout` action for this?

Reply:
I think we may be able to use actions/checkout for the ccip checkout but not the chainlink-ccip checkout because it does a shallow clone which messes with the git SHAs

Reply:
Oops ignore what I said above, I misinterpreted the context of the comment.

Reply:
Switched to actions/checkout